src/tests/test_mem_btree.cpp

```diff
@@ -424,6 +431,152 @@ TYPED_TEST(BtreeTest, RangeUpdate) {
     this->query_validate(0, num_entries - 1, 75);
 }
 
+// Comment this for now after making the decision how to treat corner cases
```

Comment:
I'd prefer to simply leave this out and put it in a new pr. Having it like this could grow stale and not even build anymore when someone wants to uncomment it versus a pr that builds and would get attention eariler?